modules/CurrentDate.js

```diff
@@ -0,0 +1,7 @@
+import { DateTime } from 'luxon';
+
+export default function getCurrentDate() {
+  const now = DateTime.now();
+  const formattedDate = now.toLocaleString(DateTime.DATETIME_FULL);
+  return formattedDate;
+}
```

Comment:
Please refactor this function to use arrow syntax to adhere to the project requirements🙏